variables.tf

```diff
@@ -68,7 +69,8 @@ variable "sa_role" {
 
 variable "apis_authority" {
   description = "Toggles authoritative management of project services."
-  default     = "false"
```

Comment:
Not related to your change, please revert

Reply:
reverted